javascript/statements.json

```diff
@@ -1326,6 +1324,91 @@
               "status": {
                 "experimental": false,
                 "standard_track": false,
+                "deprecated": true
+              }
+            }
+          }
+        },
+        "import_attributes": {
+          "__compat": {
+            "description": "Import attributes",
+            "support": {
+              "chrome": {
+                "version_added": false,
+                "impl_url": "https://crbug.com/v8/13856"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": "1.37"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
+                "impl_url": "https://bugzil.la/1736059"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "type_json": {
+            "__compat": {
+              "description": "import with {type: json}",
```

Comment:
This was a sub feature of the previous "import assertion" feature. Do we need this sub feature again for the new "import attribute"? Looks like its compat data is the same as the main "import attribute" feature...

Reply:
I think that we should keep this subfeature in, in case additional types are added in the future.

Reply:
Compat data will be different as soon as browsers ship the new syntax, and Firefox/Safari will only support `type` in `with` and not `assert`.

What we could do is to just split it in two separate things: "does this support with/assert?" "does this support `type: json`"? Rather than manually multiplying the compatibility tables.